certificate-policy.md

```diff
@@ -1597,7 +1652,7 @@ All other optional attributes, when present within the subject field, MUST conta
 By issuing a Subordinate CA Certificate, the CA represents that it followed the procedure set forth in its Certificate Policy and/or Certification Practice Statement to verify that, as of the Certificate's issuance date, all of the Subject Information was accurate.
 
 ### 7.1.5 Name constraints
-All Subordinate CA Certificates shall be Technically Constrained.
+All Subordinate CA Certificates SHALL be Technically Constrained. All OCSP Responders SHALL either be technically contrained to only assert the OCSP Signing EKU (1.3.6.1.5.5.7.3.9) or not issue SHA-1 based OCSP responses.
```

Comment:
The way this is written, it seems to **allow** SHA-1 based OCSP responses if the OCSP responder is technically constrained to the OCSP signing EKU.